pkg/TIPA.py

```
@@ -85,15 +86,27 @@ def run(self) -> None:
         self.capture_screenshots()
 
     def capture_screenshots(self) -> None:
-        # Register the keyboard event once
-        keyboard.on_press_key(key="f", callback=self.on_release)
+        with open("_internal/settings.json", "r") as settings_file:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)